src/runtime_view_proxy.js

```diff
@@ -42,7 +42,7 @@ function getHeapBlock(type, offset, length) {
   }
 }
 
-function createProxyHandler(type, heapBlocks) {
+function createProxyHandler(type, heapBlocks, byteLength) {
```

Comment:
How does this `byteLength` relate to line 51's `wasmMemory.byteLength`?

Reply:
Remove this new argument